src/chrome/content/rules/asexuality.org_login.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+	Other asexuality.org rulesets:
+		asexuality.org.xml
+
+	Certificate mismatch:
+		ru.asexuality.org
+
+	Sept 2017: This ruleset has 2 functions. First, it redirects the non-forum part of the site, located at "/", to HTTPS
+	in all browsers. The forum is located at "/en/". Second, it redirects the target URL for the forum's login form to
+	HTTPS in all browsers. Due to mixedcontent issues, the rest of the forum can't be redirected to HTTPS except in
+	mixedcontent-friendly browsers, which is handled by the other ruleset file.
+-->
+<ruleset name="Asexuality.org (partial)">
+	<target host="asexuality.org" />
+		<test url="http://asexuality.org/en/" />
+		<test url="http://asexuality.org/en/login/" />
+		<test url="http://asexuality.org/?q=about.html" />
+		<test url="http://asexuality.org/?q=contact.html" />
+	<target host="www.asexuality.org" />
+		<test url="http://www.asexuality.org/en/" />
+		<test url="http://www.asexuality.org/en/login/" />
+		<test url="http://www.asexuality.org/?q=about.html" />
+		<test url="http://www.asexuality.org/?q=contact.html" />
+
+	<exclusion pattern="^http://(www\.)?asexuality\.org/en/(?!login)" />
+		<!-- Exclude all areas of the forum except the login form target URL. -->
```

Comment:
The use of negative lookahead operator is discouraged in the [contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#regex-conventions) and I don't think it is needed anyways. Your rules are specific enough to avoid those cases.